Tayfun/#27 guest page

0972251 opened the pull request:
- kaartjes
- informatie tabel
- inlog button

Rolfazay committed:
EmployeeLogin

Rolfazay committed:
test push

Rolfazay committed:
Login + Account front end push

tobisphere committed:
Fixed Photo and Navbar

Na struggel eindelijk gefixt

svenfire99 committed:
Meeting Creation Form

Rolfazay committed:
RIP XAMPP

tobisphere committed:
Working on front-end homepage

added background possibilities and working on navbar

0972251 committed:
GuestPage

tobisphere committed:
Still working on front-end

The navbar works on a few items now and home page has some extra features

tobisphere committed:
Navbar fully functional

Everything is functional on front-end except the account button

tobisphere committed:
Working on feedback forms

Trying to implement form questions and communicate with database

Rolfazay committed:
Merge remote-tracking branch 'origin/Roby/#26-Inlogscherm' into Roby/#26-Inlogscherm

Rolfazay committed:
Merge remote-tracking branch 'origin/Sven/#1_Meeting_Crud' into Roby/#26-Inlogscherm

Rolfazay committed:
Merge remote-tracking branch 'origin/Tobias/#28HoofdpaginaUser' into Roby/#26-Inlogscherm

Rolfazay committed:
Added Nav bar

log in and Account create nav bar

tobisphere committed:
Merge remote-tracking branch 'origin/Roby/#26-Inlogscherm' into Tobias/#28HoofdpaginaUser

0972251 committed:
gastpagina

moet nog wat tekst komen om meer informatie te geven over het gebruik

0972251 committed:
gastpagina

moet nog wat tekst komen om meer informatie te geven over het gebruik

0972251 committed:
mail zender

php bestand moet gekoppeld worden aan de meets aanmak scherm html.twig

0972251 committed:
mail zender

php bestand moet gekoppeld worden aan de meets aanmak scherm html.twig

0972251 committed:
guestcontroller

0972251 committed:
guestcontroller

0972251 committed:
image fix

0972251 committed:
image fix

tobisphere committed:
Account button gaat naar login

Een paar voorbereidingen op feedback task voor database

tobisphere committed:
Setup form feedback

tobisphere committed:
Working on form

tobisphere committed:
Changing branch

svenfire99 committed:
Merge branch 'Tobias/#28HoofdpaginaUser'

svenfire99 reviewed (changes requested):
- Maak het meer Cimmeets
- Loop wat styling na
- Rebase het naar master zodat je styling kan gebruiken van Tobias zijn hoofdpages eventueel

0972251 committed:
Merge remote-tracking branch 'origin/Tayfun/#27_guestPage' into Tayfun/#27_guestPage

0972251 committed:
functional guestpage

- Added path to login
- Background added like the homepage
- navbar color is the same like login
- some videos and text added

svenfire99 committed:
Just a Few Tweaks

svenfire99 committed:
More Tweaks
